spec/ruby/core/warning/element_reference_spec.rb

```diff
@@ -8,6 +8,14 @@
     end
   end
 
+  ruby_version_is '3.3' do
+    it "returns default values for :performance category" do
+      ruby_exe('p Warning[:performance]').chomp.should == "false"
```

Comment:
This one could be just `Warning[:performance].should == false` to avoid an extra subprocess.
mspec will most likely never set `Warning[:performance]` so that's not a concern.